docs/reference/compatibility.txt

```diff
@@ -499,6 +499,15 @@ Rails Frameworks Support
 Ruby on Rails is comprised of a number of frameworks, which Mongoid attempts to
 provided compatibility with wherever possible.
 
+Though Mongoid attempts to offer API compatibility with `Active Record <https://guides.rubyonrails.org/active_record_basics.html>`_,
+frameworks that depend directly on Active Record may not work as expected when
+Mongoid is used as a drop-in replacement.
+
+.. note::
+
+  Mongoid can be used alongside other Active Record adapters within the same
```

Comment:
"Mongoid can be used alongside Active Record in the same application without issue"